### PR TITLE
DUpdate succes and message body issue solved

### PR DESCRIPTION
**Pull request by rohit-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html [hunk 1]
```diff
@@ -613,7 +617,7 @@ <h3 class="fs20 mrB2">ICICI Bank</h3>
                                     </sh-select>
                                 </div>
                                 <div class="custom-select select-caret pos-rel" *ngIf="item.otpType==='BANK'">
-                                    <input type="text" class="form-control" disabled placeholder="Registered" name="approvalUniqueName{{index}}" [(ngModel)]="item.approvalUniqueName">
+                                    <input type="text" class="form-control" disabled placeholder="Registered" name="approvalUniqueName{{index}}" value="">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.html:620]: use of this field?

Comment (rohit-walkover) [reply apps/web-giddh/src/app/settings/integration/setting.integration.component.html:620]: actually we don't need any value in this case if OTP type BANK

Comment (rohit-walkover) [reply apps/web-giddh/src/app/settings/integration/setting.integration.component.html:620]: API expect us to send null if selected BANK other than need value

Comment (ravinderwalkover) [reply apps/web-giddh/src/app/settings/integration/setting.integration.component.html:620]: Remove value attribute

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html [hunk 2]
```diff
@@ -394,117 +395,120 @@ <h2 style="float: left; font-size: 20px">ICICI Bank</h2>
...
+                                                        <p>{{option?.label}}</p>
+                                                        <p>Unverifed</p>
+                                                        <img src="assets/images/close-round.svg" class="check-search" alt="">
+                                                    </div>
+                                                </fieldset>
+                                            </ng-template>
+                                        </sh-select>
+                                    </div>
+                                    <div class="custom-select select-caret pos-rel" *ngIf="item.otpType==='BANK'">
+                                        <input type="text" class="form-control" disabled placeholder="Registered" name="approvalUniqueName{{index}}" value="">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.html:486]: Is this field required?

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html [hunk 3]
```diff
@@ -394,117 +395,120 @@ <h2 style="float: left; font-size: 20px">ICICI Bank</h2>
...
                                 </div>
                             </div>
+                        </ng-container>
+
+                        <div>
+                            <a (click)="addNewAmountRangeRow(ind+1)" class="btn btn-link pl-0">+ Add New Row</a>
                         </div>
-                    </ng-container>
-                    <div>
-                        <a href="javascript:void" (click)="addNewAmountRangeRow(ind+1)" class="btn btn-link pl-0">+ Add New Row</a>
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.html:496]: Why `href="javascript:void"` was removed?

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html [hunk 4]
```diff
@@ -394,117 +395,120 @@ <h2 style="float: left; font-size: 20px">ICICI Bank</h2>
...
-                    <div class="row">
-                        <div class="form-group">
-                            <label>&nbsp;</label>
-                            <br>
-                            <button type="button" class="btn btn-success mrR1" (click)="updateICICDetails(regAcc, ind)">Update
+                        <div class="row">
+                            <div class="form-group">
+                                <label>&nbsp;</label>
+                                <br>
+                                <button type="button" class="btn btn-success mrR1" [disabled]="!paymentFormUpdate.form.valid" (click)="updateICICDetails(regAcc, ind)">Update
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.html:507]: Incorrect spelling of **updateICICDetails**
Consider using `updateIciciDetails `

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.ts [hunk 5]
```diff
@@ -859,20 +858,29 @@ export class SettingIntegrationComponent implements OnInit, AfterViewInit {
...
+     *
+     * @param {IOption} event OTP type option event
+     * @param {number} index index number
+     * @param {boolean} isUpdate True if update mode
+     * @memberof SettingIntegrationComponent
+     */
+    public selectedOtpType(event: IOption, index: number, isUpdate: boolean, parentIndex?: number): void {
+        if (event && this.paymentFormObj && this.paymentFormObj.userAmountRanges  && !isUpdate) {
+            this.paymentFormObj.userAmountRanges[index].approvalUniqueName = '';
+        } else if (event && this.registeredAccount && this.registeredAccount[parentIndex].userAmountRanges && isUpdate) {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.ts:872]: Consider adding nullity check to `this.registeredAccount[parentIndex]`

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html (hunk 2 above)
Comment (rohit-walkover) [reply apps/web-giddh/src/app/settings/integration/setting.integration.component.html:486]: No

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html (hunk 4 above)
Comment (rohit-walkover) [reply apps/web-giddh/src/app/settings/integration/setting.integration.component.html:507]: its existing

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.ts (hunk 5 above)
Comment (rohit-walkover) [reply apps/web-giddh/src/app/settings/integration/setting.integration.component.ts:872]: Already done and its defined object no need it

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html (hunk 3 above)
Comment (rohit-walkover) [reply apps/web-giddh/src/app/settings/integration/setting.integration.component.html:496]: Done

**Pull request merged**